main.html

```diff
@@ -1,77 +1,231 @@
-<p>This is a block inside a div with an id the-id.
-Very cool. Yes.</p>
-</div>
-<pre data-code-language="" >hey = &quot;hello world&quot;, 38+4
-print(hey hello. This is a non executable block )!
-</pre>
-<div id="another-id">
-<h2>More complicated example</h2>
-</div>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">x = [1,1]
-for i in range(12):
-    x.append(x[-1] + x[-2]) 
-print(', '.join(str(y) for y in x))
-</pre>
-<h2>A Chart</h2>
-<div id="this-example">
-</div>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">%matplotlib inline
-import matplotlib.pyplot as plt
-import numpy as np
-x = np.linspace(0, 3*np.pi, 500)
-plt.plot(x, np.sin(x**2))
-plt.title('A simple chirp');
-</pre>
-<h2>Seaborn Example</h2>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">from __future__ import print_function, division
-
-%matplotlib inline
-import matplotlib.pyplot as plt
-import numpy as np
-import pandas as pd
-
-x = np.linspace(0, 10, 1000)
-plt.plot(x, np.sin(x), x, np.cos(x));
-
-import seaborn as sns
-sns.set()
-plt.plot(x, np.sin(x), x, np.cos(x));
-</pre>
-<h2>Interactive widgets</h2>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">%matplotlib inline
-from IPython.display import Image
-from IPython.html.widgets import interact
-from numpy import pi, cos, sin
-import numpy as np
-import pylab as plt
-import seaborn as sns
-sns.set()
-</pre>
-<pre data-code-language="pyt" data-executable="true" data-type="programlisting">def plot_fringe(bl_length, wavelength):
-    &quot;&quot;&quot; Plot the fringe function for a baseline (see Fig 1)
-
-    bl_length:      distance between antennas, in m
-    wavelength:     wavelength, in m
-    &quot;&quot;&quot;
-    theta = np.linspace(-np.pi, np.pi, 401)
-    l = sin(theta)
-    F = cos(2 * pi * bl_length * l / wavelength)
-    F2 = cos(2 * pi * 2 * bl_length * l / wavelength)
-
-    plt.plot(l, F, c='#cc0000', label=&quot;Baseline 1-2&quot;)
-    plt.plot(l, F2, c='#0000cc', label=&quot;Baseline 1-3&quot;)
-    plt.xlabel(&quot;$sin(\\theta)$&quot;)
-    plt.ylabel(&quot;Fringe amplitude&quot;)
-    plt.ylim(-2, 2)
-    plt.legend()
-
-f = interact(plot_fringe, bl_length=(1, 100), wavelength=(1, 100))
-&lt;/pre&gt;
-</pre>
+<h2>Introduction</h2>
+
+<p>
+  InnerSource is the methodology of applying open source practices and principles to internal software development.
+  This methodology results in shared software that is responsive and nimble to the changing needs of its many internal consumers.
+</p>
+
+<p>
+  In this Oriole, we'll define some key terms used when discussing InnerSource.
+  We'll walk through the high-level mechanics of how InnerSourcing can work and enumerate the key principles upon-which these mechanics are based.
+  Finally, we'll share other references that can help you deepen your understanding and involvement in the craft of InnerSource.
```

Comment:
Consider adding some teeth to this.  


`Finally, we'll share additional resources, so that you can get involved in the community that is forming around InnerSource.`

Reply:
Suggestions?  I think the weight of this description matches the weight of the actual additional resources.  Are you saying that the description needs to carry more weight or the resources themselves aren't massive enough to call out in this way?

Reply:
The introduction is too buzz-wordy.  These things can be more specific focused on the perspective of the person going through the oriole.  

> Walk through high-level mechanics
What does this mean?   While I agree there are mechanics I'd like something practical I can do after this oriole.  Can I identify the mechanics and patterns? Anti-patterns?  etc.. something that I from the perspective of the participant can say I can do now that I couldn't before.  

>  we'll share other references 
what references? Why do I care about references at this point? I just started looking at this document.  Isn't this going to teach me about this stuff?  Seems odd so early on to setup pointers to reference material.  How do I get involved?  What are ways to engage others in the innersource community?  This is the type of action that should be called out instead I think.

>  deepen your understanding and involvement in the craft of InnerSource
My understanding?  Ok what am I going to learn?  Also I'm involved already?  I'm already participating by watching this oriole, how can you deepen my involvement?

Reply:
Use more conversational (and specific) language.

Reply:
Don't really need to introduce the inner source commons.